DemoApp/PushNotifications.swift

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import UserNotifications
+
+final class PushNotifications: NSObject {
+    var center: UNUserNotificationCenter {
+        UNUserNotificationCenter.current()
+    }
+
+    var authorizationOptions: UNAuthorizationOptions {
+        [.alert, .sound, .badge]
+    }
+
+    var onNotificationResponse: ((UNNotificationResponse) -> Void)?
+
+    static let shared = PushNotifications()
```

Comment:
I would be avoiding singletons. Why not to have it as a property in `AppDelegate`?

Reply:
Done ;)